starks/floatingpoint.py

```diff
@@ -19,7 +19,7 @@ def num_to_binary_list(index):
 
   while index > 0:
     output_list.append(index%2)
-    index = index // 2
+    index = index // 2  
```

Comment:
Could you do a pass to remove trailing whitespace? You should be able to do a search in your editor for added trailing whitespace